### DISCUSSION TITLE
xarrayMannKendall/spyder crash

### DISCUSSION THREAD

**Discussion by Murk89** [pydata/xarray#6116] [Q&A] [answered]
Hi, 
I cloned and then installed the xarrayMannKendall from here: https://github.com/josuemtzmo/xarrayMannKendall on my uni's HPC.
Since then spyder crashes with a core dumped message by the HPC and these errors: 
.../lib/python3.7/site-packages/qtpy/_init_.py:222 
Warning: Qt5 version 5.6.2 is not supported by QtPy. To ensure your application works correctly with QtPy please upgrade to Qt5 5.9.0 or later.

Can someone help with this please? Many thanks.

**Comment by Illviljan** [comment, marked as answer]
Sounds like a spyder issue, xarray doesn't use qtpy or qt5. Here's what spyder thinks about not being careful about how you install packages:

https://www.youtube.com/watch?v=Ul79ihg41Rs

You should probably reinstall your environment.
